pillar.example

```diff
@@ -173,6 +173,7 @@ bind:
       type: master                                   # Yo don't have define zone again in available_zones.
                                                      # This feature is backward compatibile and only available in debian
       notify: false                                  # if type master you need specify notify true/false
+      managed: true                                  # Set this to false if you don't want Salt to manage this zone file
```

Comment:
@sylvainfaivre if you don't mind, can you extend the comment to something like
```
# Set this to false if you don't want Salt to manage this zone file.
# If this parameter is set to true or is not set at all, the zone will be managed through salt
```
or something that makes it clear that NOT setting the parameter equals to `true` and will be managed?

Reply:
Sure, added !

Thanks for the review.